docs/pages/guides/typescript.mdx

```diff
@@ -13,7 +13,7 @@ To get started, create a **tsconfig.json** in your project root:
 
 <Terminal cmd={['$ touch tsconfig.json']} />
 
-Running `npx expo start` will prompt you to install the required dependencies (`typescript`, `@types/react`, `@types/react-native`), and automatically configure your **tsconfig.json**.
+Running `npx expo start` will prompt you to install the required dependencies (`typescript`, `@types/react`), and automatically configure your **tsconfig.json**.
```

Comment:
```suggestion
For SDK 48 and above, running `npx expo start` will prompt you to install the required dependencies (`typescript`, `@types/react`) and automatically configure your **tsconfig.json**. For SDK 47 and below, the command will prompt to install the required dependencies (`typescript`, `@types/react`, `@types/react-native`) and configure **tsconfig.json** automatically.
```

I think we should keep the package reference for older SDK's (and react native versions).